Formula/phpunit.rb

```diff
@@ -1,8 +1,8 @@
 class Phpunit < Formula
   desc "Programmer-oriented testing framework for PHP"
   homepage "https://phpunit.de"
-  url "https://phar.phpunit.de/phpunit-8.5.2.phar"
-  sha256 "984e15fbf116a19ab98b6a642ccfc139a1a88172ffef995a9a27d00c556238f1"
+  url "https://phar.phpunit.de/phpunit-9.0.0.phar"
+  sha256 ""
```

Comment:
This field must not be empty. Homebrew needs a hash to verify the downloaded package.